app/controllers/plants_controller.rb

```diff
@@ -28,6 +28,17 @@ def create
     end
   end
 
+  def update
+    @plant = Plant.friendly.find(params[:id])
+    if @plant.update(plant_params)
+      #success
+      redirect_to @plant
+    else
+      #failure
+    end
+   
+  end
+
```

Comment:
You can remove this action for now, the ability to update should stay with  the admin. and that is available at /motor_admin

Reply:
Ok, no got it ;).